docs/shared/redis-tls.mdx

```diff
@@ -0,0 +1,16 @@
+For Redis TLS connections, you can set up a client certificate or override the root certificate authority by configuring `tls` in your router's [YAML config file](https://www.apollographql.com/docs/router/overview/#yaml-config-file). For example:
+
+```yaml title="router.yaml"
+apq:
+  router:
+    cache:
+      redis:
+        urls: [ "redis://localhost:6379" ]
```

Comment:
@Geal - I think this example URL should be updated for TLS (change to `rediss` and  use non-localhost), can you suggest one?

Reply:
something like `"rediss://redis.example.com:6379"` should work